hyper_sh/__init__.py

```diff
@@ -3,7 +3,7 @@
 import json
 import os
 
-from docker import Client as DockerClient
+from docker import APIClient as DockerClient
```

Comment:
usually you do the other way around.
try import new API, and then import old api